gulpfile.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs/promises');
 const path = require('path');
+const sirv = require('sirv-cli');
```

Comment:
Could probably switch to the `sirv` package and keep one way.

Reply:
sirv is low level http middleware. Ii does not create server and not render any logs

Reply:
Ah, you are right, I thought it did.